modules/nf-core/platypus/main.nf

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Can you change this to using a `VERSION` variable instead? This way you stay consistent along the other modules. You can see an example here: https://github.com/nf-core/modules/blob/1df904b954cadbeacf128688976ffffba25ac817/modules/nf-core/scramble/clusteranalysis/main.nf#L27

Reply:
applied